Assets/Scripts/Actors/TravelSoundPlayer.cs

```diff
@@ -0,0 +1,35 @@
+﻿using UnityEngine;
+using System.Collections;
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+public class TravelSoundPlayer : ScriptableObject
+{
+	[SerializeField] AudioClip[] _travelSounds;
+	public AudioClip[] travelSounds
+	{
+		get { return _travelSounds; }
+	}
+
+	public AudioClip GetRandomClip()
+	{
+		return _travelSounds[Random.Range( 0, _travelSounds.Length )];
+	}
+
+#if UNITY_EDITOR
+	[MenuItem( "Assets/Create/Travel Sound Player" )]
+	public static void CreateTravelSoundPlayerAsset()
```

Comment:
A better place for this method would be in `Assets/Editor/MenuItems.cs`. That's where I've been putting all the other menu item methods like this, and it would mean you don't need to wrap stuff in `#if UNITY_EDITOR`.